test/error_passing_test.go

```diff
@@ -54,20 +51,18 @@ func TestErrorPropagation(t *testing.T) {
 		resp, // response
 	)
 
-	// Type assert this to a service error
-	svcErr, ok := err.(*errors.ServiceError)
-	require.Equal(t, true, ok)
-
 	// Assert our error matches
 	require.NotNil(t, err)
-	assert.Equal(t, errorCode, svcErr.Code())
-	assert.Equal(t, errorDescription, svcErr.Description())
-	assert.Equal(t, errorDescription, svcErr.Error())
+	typhonErr := err.(*errors.Error)
```

Comment:
we should probably catch a type assertion failure here, and _require_ it was `ok`